locales/en/experimental/create-kafka-instance.json

```diff
@@ -0,0 +1,106 @@
+    },
+    "preparing": {
+      "description": "Creating prerequisite resources",
+      "title": "Preparing"
+    },
+    "provisioning": {
+      "description": "Creating Kafka instance",
+      "title": "Provisioning"
+    },
+    "title": "Creating instance",
+    "warning_or_error_title": "$t(kafka_status_warning_or_error_title)"
+  },
+  "kafka_status_warning_or_error_title": "This is taking longer than expected.",
+  "learn_about_cloud_regions": "Learn more about cloud regions",
+  "learn_about_sizes": "Learn more about sizes",
+  "message_size_value": "up to {{value}} MiB",
+  "modal_alerts": {
+    "regions_unavailable_title": "Cloud provider regions are temporarily unavailable.",
+    "regions_unavailable_message": "Try again later or <0>contact support</0>.",
+    "instance_unavailable_title": "$t(common:kafka) instances are unavailable for creation",
+    "over_quota_message": "Your organization has {{allowedStreamingUnits}} streaming units available which are already in use. To deploy a new instance, delete an existing one first or <0>contact us</0> to add more streaming units to your account.",
+    "over_quota_title": "Your organization has no streaming units remaining.",
+    "system_unavailable_message": "$t(common:unexpected_error)",
+    "system_unavailable_title": "$t(common:something_went_wrong)",
+    "trial_available_message": "The trial has a duration of {{time}} hours. If you are interested in using this service in production, go to our <0>Streams for Apache Kafka Overview</0> page to see what options are available during creation and learn how to contact us.",
+    "trial_available_title": "You can create a trial instance to evaluate this service.",
+    "trial_unavailable_message": "All available trial instances are currently in use. Try again later.",
+    "trial_unavailable_title": "$t(common:kafka) instances are unavailable for creation",
+    "trial_used_message": "You can deploy 1 instance at a time. To deploy a new instance, delete your existing one first.",
+    "trial_used_title": "You already have an instance running"
+  },
+  "no_regions_available": "No regions available",
+  "partitions_value": "up to {{value}}",
+  "quick_start_guide_message": "Need help getting started? Follow our quick start guide.",
+  "quick_start_guide_message_exp": "Follow our quick start guide.",
+  "quick_start_guide_title": "Need help getting started?",
+  "size_field_aria": "More info for Size field",
+  "size_help_content": "Size of a Kafka instance is based on streaming units. The number of streaming units defines the capacity, resources, and limits of an instance. An instance with a larger size can handle higher loads and process larger amounts of events, has more storage, and can handle more clients and connections.",
+  "standard_kafka_size_dscription": "Learn how to add streaming units to your account",
+  "standard_kafka_streaming_units": "Your organization has {{streamingUnits}} streaming units remaining",
```

Comment:
If the organization only has 1 streaming unit, how do we handle singular vs plural for translation? Do we have an option to use a string like "Your organization has 1 streaming unit remaining."? (i.e. without the "s" in "units")